resources/gen-ai/aws-text-to-sql/workflow_config.json

```diff
@@ -0,0 +1,43 @@
+    {
+    "verbose": false,
+    "semantic_layer": {
+        "strategy": "auto",
+        "config_files":{
+            "prompt_template_path": "config/knowledge_layer_prompt.json"
+        },
+        "knowledge_base": {
+            "kb_prompt_template_path": "config/kb_schema_linking_prompt.json",
+            "id": "",
+            "embedding_model": "amazon.titan-text-lite-v1"
+        },     
+        "model_id": "anthropic.claude-3-haiku-20240307-v1:0",
+        "topp": 1,
+        "topk": 1,
+        "temperature": 0
+    },
+    "sql_generation": {
+        "strategy": "auto",
+        "few_shots_file_path": "config/few_shots.json",
+        "few_shots_strategy": "static",
+        "few_shots_max": 5,
+        "model_id": "anthropic.claude-3-haiku-20240307-v1:0",
+        "topp": 1,
+        "topk": 1,
+        "temperature": 0
+    },
+    "sql_execution": {
```

Comment:
I think by default we need to disable query execution, and have customers explicitly enable it. This will ensure that people don't mistakenly perform actions against their db while testing the construct

Reply:
done - e841893e381b343d25f76b79ae4609832dfbc01c